packages/bitcore-node/src/routes/status.ts

```diff
@@ -18,6 +19,14 @@ router.get('/performance', function(_, res) {
   res.json(PerformanceTracker);
 });
 
+router.get('/:chain/:network/sync', async function(req, res) {
+  let { chain, network } = req.params;
+  const state = await StateStorage.collection.findOne({});
+  const initialSyncComplete =
+    state && state.initialSyncComplete && state.initialSyncComplete.includes(`${chain}:${network}`);
+    res.json({"initialSyncComplete": initialSyncComplete != null});
```

Comment:
I think the != null part will return true when the includes returns false.

Seems like something like this would work
```
res.json({initialSyncComplete})
```

Reply:
Hi, yes I took that case into consideration and the current implementation returns false for any non configured chains.
Whereas using `res.json({initialSyncComplete})` returns an empty `{}` for even a configured chain for me.

Reply:
Your concern is valid though and the Typescripts type system handles it. The generated javascript code for this PR is `res.json({ "initialSyncComplete": initialSyncComplete !== null });`